src/main/java/racingcar/controller/RacingCarPlay.java

```diff
@@ -0,0 +1,71 @@
+package racingcar.controller;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import racingcar.model.CarName;
+import racingcar.model.RacingCar;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+import racingcar.view.constant.ViewMessage;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static racingcar.util.Constant.*;
+
+public class RacingCarPlay {
+    InputView inputView = new InputView();
+    OutputView outputView = new OutputView();
+    public void start() {
+        List<RacingCar> racingCars = registerRacingCars();
+        int playCount = getPlayCount();
+
+        outputView.printPlayResult();
+        for (int startIndex = START_INDEX; startIndex < playCount; startIndex++) {
+            playRacing(racingCars);
+        }
+
+        showWinner(racingCars);
+    }
+
+    public List<RacingCar> registerRacingCars() {
+        return inputView.readRacingCarNames()
+                .stream()
```

Comment:
```suggestion
        return inputView.readRacingCarNames().stream()
```
요렇게 쓰는게 관례인 걸로 알고있습니당